RELEASE.md

```diff
@@ -2,6 +2,14 @@
 
 This section documents the release process. Unless you're a dangerzone developer making a release, you'll probably never need to follow it.
 
+## Large document testing
+
+Parallel to the QA process, the release candidate should be put through the large document tests in a dedicated machine to run overnight.
+
+Follow the instructions in `docs/developer/testing.md` to run the tests.
+
+These tests will identify any regressions or progression in terms of document coverage and will allow for the
```

Comment:
This sentence is cut mid-way.

Reply:
@deeplow This comment still holds, I'm afraid.

Reply:
Should now be solved in [e9882bf](https://github.com/freedomofpress/dangerzone/pull/386/commits/e9882bf2f6c683c574b445f3ddab5fea1a23a45a).